sphinx_doc/howtos/configuration/configuring-the-controller.rst

```diff
@@ -0,0 +1,318 @@
+  +----------------+----------------------------------------------------------------------------+
+  |**description** |  This parameter controls how often a new command velocity is               |
+  |                |  computed. It also controls how often the controller checks if the goal is |
+  |                |  reached. Faster robots may need to increase this value. Slower robots can |
+  |                |  use a lower value.                                                        |
+  +----------------+----------------------------------------------------------------------------+
+
+:controller_plugin_ids:
+  +----------------+----------------------------------------------------------------------------+
+  |**type**        | vector<string>                                                             |
+  +----------------+----------------------------------------------------------------------------+
+  |**default**     |  "FollowPath"                                                              |
+  +----------------+----------------------------------------------------------------------------+
+  |**description** | The name of each controller plugin to be loaded into the controller server.|
+  |                | Multiple controller plugins can be loaded simultaneously so they can share |
+  |                | a costmap instance. The name assigned to the plugin using this parameter   |
+  |                | determines how it should be addressed in the *controller_id* field of the  |
+  |                | FollowPath action request.                                                 |
+  +----------------+----------------------------------------------------------------------------+
+
+
+:controller_plugin_types:
+  +----------------+----------------------------------------------------------------------------+
+  |**type**        | vector<string>                                                             |
+  +----------------+----------------------------------------------------------------------------+
+  |**default**     |  "dwb_core::DWBLocalPlanner"                                               |
+  +----------------+----------------------------------------------------------------------------+
+  |**description** | The data type of each controller plugin to be loaded into the controller   |
+  |                | server. This is used by pluginlib to dyanmically load the object. This     |
+  |                | needs to be a class derived from nav2_core::LocalPlanner. The default      |
+  |                | controller is an instance of DWB whose parameters are here                 |
+  |                | `DWB Controller`_                                                          |
+  +----------------+----------------------------------------------------------------------------+
+
+Progress Checker
+=================
+
+This is a component of the controller server that ensures the robot is making progress towards
```

Comment:
>This is a component of the controller server that ensures the robot is making progress towards the goal.

If I am not wrong the progress checker checks only if the robot makes progress or not. It doesn't make sure if the robot moves toward any goal poses.